mmv1/products/securitycenter/EventThreatDetectionCustomModule.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'EventThreatDetectionCustomModule'
```

Comment:
Unless the folder/project variants of this resource are not possible to use, I think we want to call this `OrganizationEventThreatDetectionCustomModule`. Some similar resources are the SHA custom module ones i.e. https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/scc_organization_custom_module that should have been called `google_scc_organization_security_health_analytics_custom_module` given the EventThreatDetection modules exist too

Reply:
SGTM, this is fine then. Thanks!